waku/waku_rln_relay/rln_relay.nim

```diff
@@ -22,6 +22,7 @@ import
   ./protocol_types,
   ./protocol_metrics
 import
+  ../waku_relay, # for WakuRelayHandler
```

Comment:
not too sure if we should cause this dependency with Waku Relay, but it is tightly coupled for now